linode/region/framework_datasource_schema.go

```diff
@@ -32,6 +40,11 @@ var frameworkDataSourceSchema = schema.Schema{
 			Description: "This region’s current operational status.",
 			Computed:    true,
 		},
+		"placement_group_limits": schema.ListAttribute{
+			Description: "Information about placement groups limits for this region.",
+			Computed:    true,
+			ElementType: ObjectTypePGLimits,
+		},
```

Comment:
I was super tempted to implement this as an ObjectAttribute but decided against it to stay consistent with the rest of the provider.

Reply:
Should we build it into a nested block attribute? Because it's only for read, it doesn't seem to any blocker to do so.

Reply:
Good question! I think this is mostly a stylistic choice so it might be worth quickly discussing at Cafe tomorrow

Reply:
Going to mark this as resolved for now per the discussion in Slack. We can definitely revisit this in the future 👍